srcpkgs/ReDoomEd.app/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'ReDoomEd.app'
+pkgname=ReDoomEd.app
+version=0.92.1.b1
+revision=1
+wrksrc="ReDoomEd.Sources.${version%.*}-${version##*.}"
+build_wrksrc=ReDoomEd
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc tar which"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui gnustep-back"
+short_desc="GNUstep-based port of id Software’s Doom map editor for NeXTSTEP"
+maintainer="Kira Patton <roundduckman@protonmail.com>"
+# see LICENSE.txt and https://doomwiki.org/wiki/Data_Utility_License for more info about license,
+# this is also why the package is considered restricted, due to fuzzy legality issues
+license="AGPL-3.0-or-later, custom:id-DUL, custom:unknown"
+homepage="http://twilightedge.com/mac/redoomed/"
+distfiles="http://twilightedge.com/downloads/ReDoomEd.Sources.${version%.*}-${version##*.}.tar.gz"
+checksum=256e4e63a6f109281673fd6fb1a11d3872aea6064ae8e104cd1238c9a7c21fa5
+repository=nonfree
+restricted=yes
+
+pre_build() {
```

Comment:
What I normally do is

    if [ -e /usr/share/GNUstep/Makefiles/GNUstep.sh ];
     	. /usr/share/GNUstep/Makefiles/GNUstep.sh
    fi

Not in any particular phase.

Reply:
ok!

Reply:
so I can just put it in pre_build() or whatever I want it at?

Reply:
I do it  toplevel, not in one of the functions. I'm not sure if it is right, but it works and is less repeat-y.

Reply:
@abenson  sorry for not posting in a while, I was not doing it because of desktop configuration crap I was doing

BTW, I tried to use this if thing and it fails, even with the "then" after the semicolon of the if statement. I get the `GNUmakefile:175: /application.make: No such file or directory` error associated with the fact that the make environment setup shit wasn't executed.

Reply:
it's working now